AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -857,7 +857,7 @@ - (void)layoutSubviews
   _nextLayoutInvalidationStyle = ASCollectionViewInvalidationStyleNone;
   switch (invalidationStyle) {
     case ASCollectionViewInvalidationStyleWithAnimation:
-      if (!_superPerformingBatchUpdates) {
+      if (0 == _superBatchUpdateCount) {
```

Comment:
For BOOLs I prefer `NO == _myBool` so you can immediately see the NO. For all other assignments this "defensive ordering" with constant on the left prevents people from accidentally using a single =, so I think we should standardize on that.